src/client/pages/index.vue

```diff
@@ -18,15 +21,21 @@
 
 <script>
   export default {
-    name: 'DialogCustom',
-    data: () => ({
-      showDialog: true
-    })
+    data() {
+      return {
+        dialogIsOpen: true,
+      }
+    },
+    methods: {
+      closeDialog() {
+        this.dialogIsOpen = false
+      }
+    }
   }
 </script>
 
 <style>
-  .md-dialog {
+  .home-page__dialog{
```

Comment:
home-page__dialog suggests that this is part of a block and has no standalone meaning. However I feel it is a modifier of the vue material dialog component.

So I think it would be better to name the class: .md-dialog--w600